packages/certificator/readme.md

```diff
@@ -0,0 +1,156 @@
+# Certificator
+An [ACME](https://tools.ietf.org/html/rfc8555) client for automatically managing accounts and ssl certificates with Let's Encrypt certificate authority.
+
+## Prerequisites
+Requires node 10.x.x for updated crypto library.
+
+## Usage
+Use `npm link` to locally run the `certificator` command directly from the terminal.
+
+```
```

Comment:
Doesn't 'npm install -g' do this? I've never seen a package that requires npm link

Reply:
ah ya global install does the linking. ill change the readme.